web-app/src/pages/create-review/create-review-inner-form.tsx

```diff
@@ -23,7 +23,7 @@ const FormDividerLine = styled.hr`
 	margin-right: -30px;
 `;
 
-function InnerForm({ submissionError }) {
+function InnerForm({ submissionError }: string | null) {
```

Comment:
This type is wrong. If submission error is a string or null it should be 
```suggestion
function InnerForm({ submissionError }: { submissionError: string | null }) {
```

Reply:
Good catch, thanks